realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -38,6 +38,16 @@
     // will forward this query to the host, running the integration test server on 8888
     private final static String START_SERVER = "http://127.0.0.1:8888/start";
     private final static String STOP_SERVER = "http://127.0.0.1:8888/stop";
+    private static boolean inited = false;
+
+    // Start the ROS service only once since there are problems with reseting it. Also it makes testing faster.
```

Comment:
It might be my fault ... After clean up everything, the original solution (start/stop in BeforeClass/AfterClass) works. Will revert this change.